integration/benchnet2/automate/bootstrap_automation.go

```
@@ -0,0 +1,196 @@
+	var collectionNodes = NodesDefs{Defaults: *nodeTypeResources, Nodes: collectionNodeMap}
+
+	consensusTemplatePath := templateFolder + CONSENSUS_TEMPLATE
+	consensusNodeMap := nodeStruct("consensus", consensusTemplatePath, DEFAULT_CONSENSUS_IMAGE, nodesData, nodeConfig)
+	var consensusNodes = NodesDefs{Defaults: *nodeTypeResources, Nodes: consensusNodeMap}
+
+	executionTemplatePath := templateFolder + EXECUTION_TEMPLATE
+	executionNodeMap := nodeStruct("execution", executionTemplatePath, DEFAULT_EXECUTION_IMAGE, nodesData, nodeConfig)
+	var executionNodes = NodesDefs{Defaults: *nodeTypeResources, Nodes: executionNodeMap}
+
+	verificationTemplatePath := templateFolder + VERIFICATION_TEMPLATE
+	verificationNodeMap := nodeStruct("verification", verificationTemplatePath, DEFAULT_VERIFICATION_IMAGE, nodesData, nodeConfig)
+	var verificationNodes = NodesDefs{Defaults: *nodeTypeResources, Nodes: verificationNodeMap}
+
+	return &Values{Branch: "fake-branch", Commit: "123456", Defaults: EmptyStruct{}, Access: accessNodes, Collection: collectionNodes,
+		Consensus: consensusNodes, Execution: executionNodes, Verification: verificationNodes}
+}
+
+func nodeStruct(nodeType string, nodeTemplatePath string, imageTag string, nodesData map[string]Node, nodeConfig map[string]int) map[string]NodeDetails {
+	nodeTemplate := createTemplate(nodeTemplatePath)
+	var nodeMap = make(map[string]NodeDetails)
+
+	for i := 1; i <= nodeConfig[nodeType]; i++ {
+		name := fmt.Sprint(nodeType, i)
+		replacementData := ReplacementData{NodeID: nodesData[name].NodeID, ImageTag: imageTag}
+
+		nodeString := replaceTemplateData(nodeTemplate, replacementData)
+		nodeStruct := unmarshalToStruct(nodeString, &NodeDetails{}).(*NodeDetails)
+		nodeStruct.Image = replacementData.ImageTag
+		nodeStruct.NodeID = replacementData.NodeID
+
+		nodeMap[name] = *nodeStruct
+	}
+
+	return nodeMap
+}
+
+func textReader(path string) []byte {
+	file, err := os.ReadFile(path)
+	if err != nil {
+		log.Fatal(err)
```

We need to return the error here after logging it.